extensions/skins/conversation_v1/Conversation.js

```diff
@@ -523,6 +526,18 @@ oppia.directive('conversationSkin', [function() {
         }, true);
       };
 
+      $scope.submitAnswer = function(answer) {
+        var timeAtSubmission = new Date().getTime();
+        if (_answerIsBeingProcessed) {
+          return;
+        }
+        _answerIsBeingProcessed = true;
+        oppiaPlayerService.getClassificationResult(answer).success(function(classificationResult) {
+          _answerIsBeingProcessed = false;
```

Comment:
This reads strangely in conjunction with the following line. Maybe _answerIsBeingClassified?